ldapwatch.go

```diff
@@ -16,21 +16,14 @@ type Searcher interface {
 
 // Checker ...
 type Checker interface {
-	Check(Result)
+	Check(*ldap.SearchResult, error)
```

Comment:
This matches the return of `Search()` defined by the `ldapwatch.Searcher` interface (e.g. [`ldap.Conn.Search`](https://godoc.org/gopkg.in/ldap.v2#Conn.Search)).